code/modules/mob/living/silicon/pai/recruit.dm

```diff
@@ -36,8 +36,11 @@ var/datum/paiController/paiController			// Global handler for pAI candidates
 		if(href_list["download"])
 			var/datum/paiCandidate/candidate = locate(href_list["candidate"])
 			var/obj/item/paicard/card = locate(href_list["device"])
+			var/mob/downloader = locate(href_list["downloader"])
 			if(card.pai)
 				return
+			if(downloader.stat == DEAD || istype(downloader,/mob/dead/observer))

```

Comment:
I'm not sure as to whether this is the right check to be doing but you should use `isobserver(downloader)` instead of `istype(downloader,/mob/dead/observer`